### DISCUSSION TITLE
Folders

### DISCUSSION THREAD

**Discussion by demanleq** [iSoron/uhabits#1871] [Feature Requests]
I have about a thousand different habits. Their execution became impossible due to the constant monotonous scrolling up and down. Creating folders into which you can divide screens with habits by category (as in telegram) would make this application perfect.

**Comment by adrienbeau** [comment]
I cannot understand how you can literally have one thousand _habits_. Are you sure you have not turned the app into some kind of to-do list?

Even for a to-do list, one thousand open tasks is overwhelming.

As a reminder, here's the definition of habits: _a settled or regular tendency or practice, especially one that is hard to give up_.

**Reply by demanleq** [reply]
At the moment, as far as I know, I have about 870 habits, about 100 of them relate to sports, about 150 care nutrition, also learning, leisure, hygiene, social relations, and so on. I'm just very painstaking, I like this app, it's exactly what I was looking for, it has significantly changed my life, made it better and happier, I was able to become the best version of myself and continue to do it, but due to the fact that there is no quick search, sorting by tags, folders or according to the subtasks, it has turned into a huge array of poorly structured data, as if all Windows files are in one folder from system to photos with cats.

**Comment by ezio-melotti** [comment]
See these feature requests:
* https://github.com/iSoron/uhabits/discussions/495
* https://github.com/iSoron/uhabits/discussions/31
